de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/shipmentschedule/api/IInOutProducerFromShipmentScheduleWithHU.java

```diff
@@ -74,4 +73,13 @@
 	 * @see I_M_InOutLine#setIsManualPackingMaterial(boolean)
 	 */
 	IInOutProducerFromShipmentScheduleWithHU setManualPackingMaterial(boolean manualPackingMaterial);
+
+	/**
+	 * If the flag IsShipmentDateToday is true, the shipment will be created for today, no matter what delivery dates are set in the shipment schedules.
+	 * Otherwise, the date on the shipment will be the minimum date of the shipment schedules, not older than today.
+	 * 
+	 * @param isShipmentDateToday
+	 * @return
+	 */
+	IInOutProducerFromShipmentScheduleWithHU setShipmentDateToday(boolean isShipmentDateToday);
```

Comment:
suggestion: call it `computeShipmentDate(boolean forceDateToday)`

Reply:
Done